

0it [00:00, ?it/s][A[A0 , Avg Rew 0.0
Exception ignored in: <function tqdm.__del__ at 0x7f8ed8482310>
Traceback (most recent call last):
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/tqdm/std.py", line 1145, in __del__
    self.close()
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/tqdm/std.py", line 1287, in close
    fp_write('')
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/tqdm/std.py", line 1284, in fp_write
    self.fp.write(_unicode(s))
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/tqdm/utils.py", line 142, in inner
    return func(*args, **kwargs)
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 100, in new_write
    cb(name, data)
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 796, in _console_callback
    self._backend.interface.publish_output(name, data)
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 187, in publish_output
    self._publish_output(o)
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 192, in _publish_output
    self._publish(rec)
  File "/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 518, in _publish
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown

0it [00:00, ?it/s][A<class 'list'>



0it [00:00, ?it/s][A[A[A/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/numpy/core/fromnumeric.py:3372: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/shervinazadi/opt/miniconda3/envs/mass_dqn/lib/python3.8/site-packages/numpy/core/_methods.py:170: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
0 , Avg Rew nan , Avg Len nan , Episodes 0.0




0it [00:00, ?it/s][A[A[A[A0 , Avg Rew nan , Avg Len nan , Episodes 0.0
0it [00:20, ?it/s]
